fcc-js/es6/compare-scopes-of-the-var-and-let-keywords/README.md

```diff
@@ -1 +1,75 @@
-# Add challenge details and instructions here
+# Compare Scopes of the var and let Keywords
+
+If you are unfamiliar with `let`, check out [this challenge about the difference between `let` and `var`](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/explore-differences-between-the-var-and-let-keywords).
+
+When you declare a variable with the `var` keyword, it is declared globally, or locally if declared inside a function.
+
+The `let` keyword behaves similarly, but with some extra features. When you declare a variable with the `let` keyword inside a block, statement, or expression, its scope is limited to that block, statement, or expression.
+
+For example:
+
+```
```

Comment:
For all js code snippets, please add js in front. That is ```js

This is with js:
```js
var numArray = [];
for (var i = 0; i < 3; i++) {
  numArray.push(i);
}
console.log(numArray);
console.log(i);
```

This is without js:
```
var numArray = [];
for (var i = 0; i < 3; i++) {
  numArray.push(i);
}
console.log(numArray);
console.log(i);
```

You can see the syntax highlighting.

Reply:
I have make the necessary changes sir, and I have commit and push it sir